documentation/course/e2/README.md

```diff
@@ -1798,6 +1932,21 @@ TODO @tchojnacki: Dodać diagram pakietów, opis architektury.
 
 TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
+### API
+
+| **Rola**    | **Metoda** | **Endpoint**                   | **Wymagania** | **Opis**                                                                                                                |
+| ----------- | ---------- | ------------------------------ | ------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `guest`     | `GET`      | `/ext/v1/offers`               | TODO          | Pobranie listy dostępnych ofert biletowych.                                                                             |
+| `guest`     | `GET`      | `/ext/v1/offers/:id`           | TODO          | Pobranie informacji o ofercie biletowej.                                                                                |
+| `passenger` | `GET`      | `/ext/v1/tickets`              | TODO          | Pobranie listy zakupionych biletów.                                                                                     |
+| `passenger` | `POST`     | `/ext/v1/tickets`              | TODO          | Zakup biletu.                                                                                                           |
+| `passenger` | `GET`      | `/ext/v1/tickets/:id`          | TODO          | Pobranie informacji o bilecie.                                                                                          |
+| `passenger` | `POST`     | `/ext/v1/tickets/:id/validate` | TODO          | Walidacja biletu.                                                                                                       |
+| `inspector` | `POST`     | `/ext/v1/tickets/:id/inspect`  | TODO          | Inspekcja biletu.                                                                                                       |
+| `admin`     | `POST`     | `/ext/v1/offers`               | TODO          | Utworzenie nowej oferty biletowej.                                                                                      |
```

Comment:
**nit:** How about "oferta biletu" instead of "oferta biletowa"? To me it sounds like "oferta biletowa" refers to plural instead of singular thing (as if it created multiple `TicketOffer`s).